### PR TITLE
Updated index.html with fan page info

### PR DESCRIPTION
**Pull request by JonMacRae**

### CODE REVIEW

File: fan-page [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 76eff47ae299e80e9ce857735b168be1e079a72c
```
Comment (mbMosman) [inline fan-page]: I am wondering if this is why your page will not come up live on GitHub...  Make sure to always check that as part of pushing up your changes.  If you're not able to see it, you can always contact me for help if you aren't sure what is going on.  Often though you will get an email from GitHub saying your build failed with an error message.  That message though is sometimes a bit cryptic.

File: index.html [hunk 2]
```diff
@@ -1 +1,49 @@
 
+<!DOCTYPE html>
+<html lang="en">
+<head> 
+<Title> The Golf Experience</title>
```
Comment (mbMosman) [inline index.html]: It is a best practice to keep all of your tags lowercase.

File: index.html [hunk 3]
```diff
@@ -1 +1,49 @@
...
+<meta charset="utf-8">
```
Comment (mbMosman) [inline index.html]: Title and charset? :heavy_check_mark:

File: index.html [hunk 4]
```diff
@@ -1 +1,49 @@
...
+</head>
+<Body>
+	<header> <h1> The Golf Experience</h1></header>
```
Comment (mbMosman) [inline index.html]: Header?  :heavy_check_mark:

File: index.html [hunk 5]
```diff
@@ -1 +1,49 @@
...
+	  <main><h2>The Experience of Golf
+		</h2>
+		<p>
+		 The game of golf is as polorizing as any! Golf provides some 
+		 with a relaxig pass time while driving others to act out in fits of rage.
+		 It is that polorizationthat makes the experience of golf such a fantastic topic. We will explore the good and the bad in more detail as we go.
+		</p>
+		 
+		<h3> The Bad
+		</h3>
```
Comment (mbMosman) [inline index.html]: At least two subheadings?  :heavy_check_mark:

File: index.html [hunk 6]
```diff
@@ -1 +1,49 @@
...
+		 <p> Before we get into the beauty of the game, let's take a moment to appreciate the more entertaining list of the worst people to golf with.
+		 </p>
+		<ol>
+		  <li>The <em>I didn't pass math class and can't keep track of my score </em>person.</li>
+		  <li>The <em>I think I am better than I am </em>person.</li>
+		  <li>The <em>always finds his ball guy</em> even thought it went in the pond.</li>
+		  <li>The <em> sandbaging son of a bitch.</em></li>
+		  <li>And fianlly we have the <em>footwedge</em> guy who always improves his lie when no one is looking.</li>
+		</ol>
```
Comment (mbMosman) [inline index.html]: A list of something? :heavy_check_mark:

File: index.html [hunk 7]
```diff
@@ -1 +1,49 @@
...
+  		<dd>This is the Quintessential imige of a domminent golfer, imperessing your friends and intimidating your opponents!</dd>
+ 		<dt><strong>The well struck Iron:</strong></dt>
+ 		<dd>The mark of an advanced golfer. These shots may not be as impressive as the 300 yard drive, but they do more for your score card and in the end they don't ask how you did it.  The feeling of hitting a green from 200 yards is second to none. </dd>
+		<dt><strong>The 20 foot putt:</strong></dt>
+ 		<dd> While this is clearly the most dule of of the three to watch, Putting does more for your score card than any other shot. The mature golfer knows that these putts mean dollars in their pockets!</dd>
+		</dl>
+</main>	
+<footer>
+<small><i>Copright &copy; 2016 Jon MacRae | All rights have been obtained for site materieal
+<br>Email: <a href="mailto:jonmacrae@golf.com?Subject=golf" target="_top">Jon MacRae</a></i></small></Footer>
```
Comment (mbMosman) [inline index.html]: Footer with copyright and email?  :heavy_check_mark:

**Comment by mbMosman** [comment]
Good job!  All HTML is valid and you've got a good start on your page.  Check the notes for a few things to fix and make sure that the page will display on GitHub for part 2.

**Comment by JonMacRae** [comment]
I am sorry this is a very delayed response on this. I have a pretty good idea what's going on, I was struggling to with the git init command and then understanding the branch concept from there. I will take care of this before I submit again this week. Also I have fixed the capitalized tag, so should be good thanks for the feedback!

**Pull request closed without merging**